### PR TITLE
Implement ListVotes(), DeleteVotes() and add missing tests

### PR DESCRIPTION
**Pull request by opalmer**

### CODE REVIEW

**Comment by opalmer** [comment]
@andygrunwald PTAL once this merges I think we can release 0.5.2.

**Comment by andygrunwald** [comment]
LGTM. Good work!
Impressive how this lib grew since you are contributing. Thanks a log.

File: changes_reviewer.go [hunk 1]
```diff
@@ -25,6 +25,15 @@ type AddReviewerResult struct {
 	Confirm   bool           `json:"confirm,omitempty"`
 }
 
+// DeleteVoteInput  entity contains options for the deletion of a vote.
```
Comment (shurcooL) [inline changes_reviewer.go]: Minor nit, there are 2 spaces before "entity".

Comment (opalmer) [reply changes_reviewer.go]: Thanks! Fixed in 95b11af

**Pull request merged**